composer.json

```diff
@@ -19,7 +19,7 @@
         "php": ">=7.1.3",
         "guzzlehttp/guzzle": "7.*",
         "intergo/sms.to-php": "^1.0",
-        "illuminate/support": "^8.75 || ^9.24"
+        "illuminate/support": "^8.75 || ^10.0"
```

Comment:
Thanks for the Pull request and the fix. In order to keep also support for Laravel 9, we pushed now a new tag 1.2.0 that supports Laravel 8,9 and 10

Reply:
worked perfectly, thank you very much

Reply:
We thank you for notifying us :)